### ISSUE TITLE
The "Conduct Summary" tool does not work correctly on my Dota 2 account

### ISSUE THREAD

**Issue by Lobo-DEV** [ValveSoftware/Dota2-Gameplay#9767]
This is something that I have been seeing for a year or so, although I always excused it by saying that it would be something that was beyond my understanding; but it has reached a point where I have no doubt... the "Conduct Summary" tool does not work correctly on my Dota 2 account.

I have compared my "Conduct Summaries" with those of many colleagues in recent months, and most of them have worse summaries than mine. While they manage to reach a "Conduct" level of 10,000 and maintain it without many issues, mine keeps dropping and fluctuating, and for about a year now, I haven't come anywhere close to a "Conduct" score of 10,000.

It is true that I went through a phase where I complained about other people in chat, but that happened years ago. In the past few months, during which I've been conducting tests, I haven't even used the chat. I don't write or say anything negative to anyone; I always focus on my team and help everyone, especially since I usually play as an Off-Support or Hard Support because my preferred position of Hard Carry is already taken in 99% of the matches. I don't want to spend 30 minutes in the queue ¬¬'. I am consistently recognized as a "Featured Player" in 87% of my matches, and I am even the MvP in almost 35% of them. I NEVER receive any reports of any kind, yet my "Conduct" score keeps decreasing and moving without being able to increase. Sometimes, I receive +20 positive comments and no reports, but my "Conduct" score only goes up by 50 points or even drops by 100 points ¬¬'. It makes no sense at all.

To make matters worse, sometimes my "Conduct Summary" updates more than once in the same day without having played or even come close to 15 matches between summaries, and this is something you will see in some of the screenshots I will upload.

This is not normal, it makes no sense, and I request a review of my account. A review that Steam itself has already denied, attributing everything to the fact that "Behavior Summaries" are generated automatically... as if being "automated" makes it perfect and devoid of any issues. When did humans give so much power to technology that they don't even bother to check for irregularities? Have we gone mad, or stupid, or something worse... useless?

I want my account to be reviewed because what I'm experiencing is far from normal. And I want to point out that I have seen dozens of people facing the same issue, complaining since 2019 without receiving any help or support from Valve, which is lamentable for a company of its size.

Example Match ID (and possibly Timestamp)
No response

Screenshots
![(1) 22-05-2023](https://github.com/ValveSoftware/Dota2-Gameplay/assets/134793523/0839c45b-cc4e-4801-86f6-e07f6ac9aff9)
![(2) 29-05-2023](https://github.com/ValveSoftware/Dota2-Gameplay/assets/134793523/01d7d1aa-11e2-4d34-b563-7364ee2668d8)
![(3) 03-06-2023](https://github.com/ValveSoftware/Dota2-Gameplay/assets/134793523/56a1aa27-66a4-4e89-8fb3-9ebdf1c39997)
![(4) 08-06-2023](https://github.com/ValveSoftware/Dota2-Gameplay/assets/134793523/d0592314-f005-4c3b-bff0-b287d9d52e76)
![(5) 11-06-2023](https://github.com/ValveSoftware/Dota2-Gameplay/assets/134793523/31ee8b1d-4a01-489c-944f-fb4855e6250c)
![(6) 11-06-2023](https://github.com/ValveSoftware/Dota2-Gameplay/assets/134793523/c480c5a6-3271-4073-8865-b938d7c99f9c)
![(7) 13-06-2023](https://github.com/ValveSoftware/Dota2-Gameplay/assets/134793523/257ea5fa-593a-45c5-b1b8-02d3bf3533eb)
![(8) 17-06-2023](https://github.com/ValveSoftware/Dota2-Gameplay/assets/134793523/a4a986a8-b0b7-44ac-9734-5cf5de49fc5b)

**Comment by Avarise831** [comment]
<3 86% means u get 1-2 reports but it doesn't show up (yes it's confusing)
Advice: play turbo at least until get 8000 behavior score
commend give you almost nothing, you need just can't get reports

**Comment by Avarise831** [comment]
<3 86% means u get 1-2 reports but it doesn't show up (yes it's confusing)
Advice: play turbo at least until get 8000 behavior score
positive comments do not give anything they are useless

**Comment by Avarise831** [comment]
<3 86% means u get 1-2 reports but it doesn't show up (yes it's confusing)
Advice: play turbo at least until get 8000 behavior score
positive comments do not give anything just useless

**Comment by Lobo-DEV** [comment]
> <3 86% means u get 1-2 reports but it doesn't show up (yes it's confusing) Advice: play turbo at least until get 8000 behavior score positive comments do not give anything they are useless

I understand that you interpret that, but in Dota 2 it NEVER says "0 complaints", they always put "<3 complaints", even if you don't have any. It's something I've also seen many times over the years in this game and with "Behavior Summaries" from many other players who also play. I have come to play 15 full premade games in normal mode and losing the games on purpose, to guarantee that I did not receive a single repport, and even so it ALWAYS says "<3", it never says "0".

On the other hand, even if what you say was true, which in my case it is not, very strange things can be seen... like the fact that on the 11th I received 2 reports in barely 10 hours, where I have not played 15 games and that is see clearly in my history, and yet they deduct 1,500 "Conduct" points, no more, no less... for no reason because I haven't even been playing.
And you see other days in which despite having a lot of positive comments and no repports, my score keeps going down and down.

There is no excuse, this is not working properly and it is pretty obvious. Even more so when the internet is full of people complaining about this same problem in the forums for about 4 years...

**Comment by Avarise831** [comment]
> > <3 86% means u get 1-2 reports but it doesn't show up (yes it's confusing) Advice: play turbo at least until get 8000 behavior score positive comments do not give anything they are useless
> 
> I understand that you interpret that, but in Dota 2 it NEVER says "0 complaints", they always put "<3 complaints", even if you don't have any. It's something I've also seen many times over the years in this game and with "Behavior Summaries" from many other players who also play. I have come to play 15 full premade games in normal mode and losing the games on purpose, to guarantee that I did not receive a single repport, and even so it ALWAYS says "<3", it never says "0".
> 
> On the other hand, even if what you say was true, which in my case it is not, very strange things can be seen... like the fact that on the 11th I received 2 reports in barely 10 hours, where I have not played 15 games and that is see clearly in my history, and yet they deduct 1,500 "Conduct" points, no more, no less... for no reason because I haven't even been playing. And you see other days in which despite having a lot of positive comments and no repports, my score keeps going down and down.
> 
> There is no excuse, this is not working properly and it is pretty obvious. Even more so when the internet is full of people complaining about this same problem in the forums for about 4 years...

I'm dota2 player just like you and have the same problem. My advice is just don't enter ranked play turbo mode until your score reaches 8000

**Comment by Lobo-DEV** [comment]
> > > <3 86% significa que obtienes 1-2 informes pero no aparece (sí, es confuso) Consejo: juega turbo al menos hasta obtener 8000 puntaje de comportamiento comentarios positivos no den nada son inútiles
> > 
> > 
> > Entiendo que interpretes eso, pero en Dota 2 NUNCA dice "0 quejas", siempre ponen "<3 quejas", aunque no tengas ninguna. Es algo que también he visto muchas veces a lo largo de los años en este juego y con los "Resúmenes de comportamiento" de muchos otros jugadores que también juegan. He llegado a jugar 15 juegos prefabricados completos en modo normal y perdiendo los juegos a propósito, para garantizar que no recibí ni un solo reporte, y aun así SIEMPRE dice "<3", nunca dice "0".
> > Por otro lado, aunque fuera cierto lo que dices, que en mi caso no lo es, se ven cosas muy raras... como que el día 11 he recibido 2 informes en apenas 10 horas, donde no he jugué 15 partidos y eso se ve claro en mi historial, y sin embargo me restan 1.500 puntos de "Conducta", ni más, ni menos... sin razón porque ni siquiera he estado jugando. Y ves otros días en los que a pesar de tener muchos comentarios positivos y ningún reporte, mi puntaje sigue bajando y bajando.
> > No hay excusa, esto no está funcionando correctamente y es bastante obvio. Más aún cuando internet está lleno de gente quejándose de este mismo problema en los foros desde hace como 4 años...
> 
> Soy jugador de dota2 como tú y tengo el mismo problema hace 1 año. Mi consejo es que no entres en modo clasificado, juega en modo turbo hasta que tu puntuación llegue a 8000.

Yes... you've already told me that, but I'm not looking to solve this with a poor band-aid, what I think is that Valve should take this seriously, review my account and its "Conduct" system and apply the necessary solutions because it's a real problem.

It is not something that players should """solve""" by playing Turbo Mode, it is something serious that Valve must solve as a company that it is.

**Comment by Lobo-DEV** [comment]
> > > <3 86% means u get 1-2 reports but it doesn't show up (yes it's confusing) Advice: play turbo at least until get 8000 behavior score positive comments do not give anything they are useless
> > 
> > 
> > I understand that you interpret that, but in Dota 2 it NEVER says "0 complaints", they always put "<3 complaints", even if you don't have any. It's something I've also seen many times over the years in this game and with "Behavior Summaries" from many other players who also play. I have come to play 15 full premade games in normal mode and losing the games on purpose, to guarantee that I did not receive a single repport, and even so it ALWAYS says "<3", it never says "0".
> > On the other hand, even if what you say was true, which in my case it is not, very strange things can be seen... like the fact that on the 11th I received 2 reports in barely 10 hours, where I have not played 15 games and that is see clearly in my history, and yet they deduct 1,500 "Conduct" points, no more, no less... for no reason because I haven't even been playing. And you see other days in which despite having a lot of positive comments and no repports, my score keeps going down and down.
> > There is no excuse, this is not working properly and it is pretty obvious. Even more so when the internet is full of people complaining about this same problem in the forums for about 4 years...
> 
> I'm dota2 player just like you and have the same problem 1 year ago. My advice is just don't enter ranked,play turbo mode until your score reaches 8000

Yes... you've already told me that, but I'm not looking to solve this with a poor band-aid, what I think is that Valve should take this seriously, review my account and its "Conduct" system and apply the necessary solutions because it's a real problem.

It is not something that players should """solve""" by playing Turbo Mode, it is something serious that Valve must solve as a company that it is.

**Comment by Avarise831** [comment]
> 

everything is fine with your account, you are consistently getting 1-2 reports and it is displayed as <3 and you get - a behavior score. I do not understand how you can have such low decency I am not your psychiatrist and will not solve your mental problems. Good luck

**Comment by Avarise831** [comment]
> > > > <3 86% means u get 1-2 reports but it doesn't show up (yes it's confusing) Advice: play turbo at least until get 8000 behavior score positive comments do not give anything they are useless
> > > 
> > > 
> > > I understand that you interpret that, but in Dota 2 it NEVER says "0 complaints", they always put "<3 complaints", even if you don't have any. It's something I've also seen many times over the years in this game and with "Behavior Summaries" from many other players who also play. I have come to play 15 full premade games in normal mode and losing the games on purpose, to guarantee that I did not receive a single repport, and even so it ALWAYS says "<3", it never says "0".
> > > On the other hand, even if what you say was true, which in my case it is not, very strange things can be seen... like the fact that on the 11th I received 2 reports in barely 10 hours, where I have not played 15 games and that is see clearly in my history, and yet they deduct 1,500 "Conduct" points, no more, no less... for no reason because I haven't even been playing. And you see other days in which despite having a lot of positive comments and no repports, my score keeps going down and down.
> > > There is no excuse, this is not working properly and it is pretty obvious. Even more so when the internet is full of people complaining about this same problem in the forums for about 4 years...
> > 
> > 
> > I'm dota2 player just like you and have the same problem 1 year ago. My advice is just don't enter ranked,play turbo mode until your score reaches 8000
> 
> Yes... you've already told me that, but I'm not looking to solve this with a poor band-aid, what I think is that Valve should take this seriously, review my account and its "Conduct" system and apply the necessary solutions because it's a real problem.
> 
> It is not something that players should """solve""" by playing Turbo Mode, it is something serious that Valve must solve as a company that it is.

everything is fine with your account, you are consistently getting 1-2 reports and it is displayed as <3 and you get - a behavior score. I do not understand how you can have such low decency I am not your psychiatrist and will not solve your mental problems Good luck

**Comment by Lobo-DEV** [comment]
> > > > > <3 86% means u get 1-2 reports but it doesn't show up (yes it's confusing) Advice: play turbo at least until get 8000 behavior score positive comments do not give anything they are useless
> > > > 
> > > > 
> > > > I understand that you interpret that, but in Dota 2 it NEVER says "0 complaints", they always put "<3 complaints", even if you don't have any. It's something I've also seen many times over the years in this game and with "Behavior Summaries" from many other players who also play. I have come to play 15 full premade games in normal mode and losing the games on purpose, to guarantee that I did not receive a single repport, and even so it ALWAYS says "<3", it never says "0".
> > > > On the other hand, even if what you say was true, which in my case it is not, very strange things can be seen... like the fact that on the 11th I received 2 reports in barely 10 hours, where I have not played 15 games and that is see clearly in my history, and yet they deduct 1,500 "Conduct" points, no more, no less... for no reason because I haven't even been playing. And you see other days in which despite having a lot of positive comments and no repports, my score keeps going down and down.
> > > > There is no excuse, this is not working properly and it is pretty obvious. Even more so when the internet is full of people complaining about this same problem in the forums for about 4 years...
> > > 
> > > 
> > > I'm dota2 player just like you and have the same problem 1 year ago. My advice is just don't enter ranked,play turbo mode until your score reaches 8000
> > 
> > 
> > Yes... you've already told me that, but I'm not looking to solve this with a poor band-aid, what I think is that Valve should take this seriously, review my account and its "Conduct" system and apply the necessary solutions because it's a real problem.
> > It is not something that players should """solve""" by playing Turbo Mode, it is something serious that Valve must solve as a company that it is.
> 
> everything is fine with your account, you are consistently getting 1-2 reports and it is displayed as <3 and you get - a behavior score. I do not understand how you can have such low decency I am not your psychiatrist and will not solve your mental problems Good luck

I definitely don't believe that you have the ability to solve anything at all... But thank you for your repetitive and unsubstantiated responses, and thank you for not resolving anything.
What I still don't understand is who asked you in the first place... as this post is intended for the Dota 2 staff, not for you.

**Comment by BenLubar** [comment]
Behavior score is not solely based on the information in your conduct summary.

**Comment by Lobo-DEV** [comment]
> Behavior score is not solely based on the information in your conduct summary.


I know, but there's no other factor working against me, because as I mentioned before, I conducted several tests, including playing full premade with a 5-person team, so I know my team won't report me. I also played matches in Normal Mode and intentionally lost them subtly so that the enemy team wouldn't have reasons to report us either. I've played many matches like this and I've realized that even if everything is positive in my last 15 matches, my score keeps dropping and dropping.

And I repeat once again, it's not just happening to me, it's a widespread issue that affects hundreds of players, and it's something that can be easily seen in Dota 2 discussions. Other players who have also been conducting tests share images of their senseless Conduct Summary. And apart from this, what sense does it make for me to receive TWO Conduct Summary updates on the same day with only a 10-hour difference, suddenly deducting 1,500 points, when in my last 15 matches I've had very positive comments, no abandons, and no reports? Moreover, I haven't played another 15 matches for it to update again; I've only played 2 matches in that time... What's the logic behind this? It's not the first time it has happened either.

The Conduct System is FLAWED, it has ERRORS that need to be corrected. Why do they make so many excuses to deny it? It's not like Valve is paying them a monthly salary to speak well of them... When they do things right, I'm the first one to applaud them, but when they do things wrong, they also deserve criticism so that they can fix things.
In the past two decades, it seems like video game companies have turned into political parties or football teams... No matter how badly they do things, there are always people defending them tooth and nail, coming up with thousands of excuses for everything, as if by defending them, they'll be rewarded with something. The Conduct System is FAILING, that's all, and it's not that hard to realize when you see the hundreds of people complaining about the same problem and posting images of their Conduct Summaries, just like I'm doing.

**Comment by BenLubar** [comment]
I've looked at the public chat logs for some of your recent matches:

https://stratz.com/matches/7204777557/log
https://stratz.com/matches/7204487987/log
https://stratz.com/matches/7204116753/log

I am no more knowledgeable about how behavior scores are calculated than anyone else who plays the game, but assuming it looks for patterns of behavior in the same way that other similar systems do, you might want to cut back on the all caps all-chatting.

I'm not saying that turning caps lock off will result in an immediate increase of your behavior score (and again, I'm only able to hypothesize here), but in general try to avoid doing things that look similar to things someone worth reporting would do.

For example (and I'm not accusing you of doing any of these), don't buy back and run down mid lane while most of your team is somewhere else. Don't spam pings. Don't put items on the ground.

The system isn't going to penalize you for doing stuff by accident (I've definitely put multiple wards in the same place by accident in quite a few matches and I'm still 10k behavior score), but stuff that looks like bad manners will probably lower your behavior score.

**Comment by Lobo-DEV** [comment]
I don't leave items on the ground, nor do I ping in a spammy manner, nor do I usually go alone because, as I mentioned, I usually play some kind of support (since my natural position as a core is usually taken by someone else and I don't want long queues). However, I will still keep all of this in mind, thank you.

One thing I do is leave a message at the end of each match, which usually says:
"GGWP. SWAP COMMENDS <Hero Name>", and I always write it in uppercase. I don't believe that this is considered spam or misconduct due to the use of uppercase letters, but I will take it into account from now on and stop writing it in uppercase. Thank you for pointing that out as well.

Taking all of this into account, I still see that many things in my "Conduct Summaries" don't make sense, such as receiving multiple summaries in the same day, deducting 1,500 conduct points, and it's not the first time it has happened.
I also don't understand how I can have teammates who are toxic, who spend every single match insulting others, receive reports in their "Conduct Summaries," have fewer commendations than me, and even have abandons, yet they have better "Conduct" than me. It's something I can't wrap my head around.

I will stop sending the message "GGWP. SWAP COMMENDS <Hero Name>" at the end of matches, and I will continue to pay attention to all of this because it's not normal.

**Comment by Lobo-DEV** [comment]
> I've looked at the public chat logs for some of your recent matches:
> 
> https://stratz.com/matches/7204777557/log https://stratz.com/matches/7204487987/log https://stratz.com/matches/7204116753/log
> 
> I am no more knowledgeable about how behavior scores are calculated than anyone else who plays the game, but assuming it looks for patterns of behavior in the same way that other similar systems do, you might want to cut back on the all caps all-chatting.
> 
> I'm not saying that turning caps lock off will result in an immediate increase of your behavior score (and again, I'm only able to hypothesize here), but in general try to avoid doing things that look similar to things someone worth reporting would do.
> 
> For example (and I'm not accusing you of doing any of these), don't buy back and run down mid lane while most of your team is somewhere else. Don't spam pings. Don't put items on the ground.
> 
> The system isn't going to penalize you for doing stuff by accident (I've definitely put multiple wards in the same place by accident in quite a few matches and I'm still 10k behavior score), but stuff that looks like bad manners will probably lower your behavior score.

I don't leave items on the ground, nor do I ping in a spammy manner, nor do I usually go alone because, as I mentioned, I usually play some kind of support (since my natural position as a core is usually taken by someone else and I don't want long queues). However, I will still keep all of this in mind, thank you.

One thing I do is leave a message at the end of each match, which usually says:
"GGWP. SWAP COMMENDS ", and I always write it in uppercase. I don't believe that this is considered spam or misconduct due to the use of uppercase letters, but I will take it into account from now on and stop writing it in uppercase. Thank you for pointing that out as well.

Taking all of this into account, I still see that many things in my "Conduct Summaries" don't make sense, such as receiving multiple summaries in the same day, deducting 1,500 conduct points, and it's not the first time it has happened.
I also don't understand how I can have teammates who are toxic, who spend every single match insulting others, receive reports in their "Conduct Summaries," have fewer commendations than me, and even have abandons, yet they have better "Conduct" than me. It's something I can't wrap my head around.

I will stop sending the message "GGWP. SWAP COMMENDS " at the end of matches, and I will continue to pay attention to all of this because it's not normal.

**Comment by commentator24** [comment]
just wondering, do you have a habit of disconnecting before game ends? like pressing that green disconnect button (not leaving the game, just disconnecting) to queue faster?

**Comment by Lobo-DEV** [comment]
> just wondering, do you have a habit of disconnecting before game ends? like pressing that green disconnect button (not leaving the game, just disconnecting) to queue faster?

No, the truth is that I always wait for the transition with a black screen, and I also review my matches after they end. I look at my KDA, GPM, XPM, how much money I spent on support items if that was my role, how much damage I dealt and received, how many camps I stacked and allied creeps I denied, and even the minute I bought the items and if it was a smart choice considering the game at that time.

Let's say that after each match, I usually spend a few minutes to see if I did well, if I did poorly, and how to improve. If you look at my profile, you can see that I currently have 4,709 matches in my profile, out of which only 61 are abandonments, approximately 1.30%, and none of them were due to anger or something similar. It's always because I'm called to do some important work or something urgent comes up that I can't postpone, and some are also because sometimes it rains heavily where I live and the internet simply goes out for a while, or the power goes out completely.
